pkg/reporter/json_reporter.go

```diff
@@ -64,6 +73,12 @@ func (jr JsonReporter) Print(reports []Report) error {
 		return err
 	}
 
-	fmt.Println(string(jsonBytes))
+	jsonBytes = append(jsonBytes, '\n')
+	fmt.Print(string(jsonBytes))
+
+	if jr.outputDest != "" {
```

Comment:
Where does outputDest get initialized to an empty string?

Reply:
[This line](https://github.com/Boeing/config-file-validator/blob/d49614c2fd63231b4e3e07eca4d9d6882fba1c4c/cmd/validator/validator.go#L75) initializes outputPtr variable as an empty string  when `-output` flag is not provided. When `reporter` is initialized, outputPtr value is assigned to the reporter filed `outputDest`.